puppet/modules/quickstack/manifests/cinder_volume.pp

```diff
@@ -139,48 +164,48 @@
-        netapp_login             => $netapp_login[0],
-        netapp_password          => $netapp_password[0],
-        netapp_server_port       => $netapp_server_port[0],
-        netapp_storage_family    => $netapp_storage_family[0],
-        netapp_transport_type    => $netapp_transport_type[0],
-        netapp_storage_protocol  => $netapp_storage_protocol[0],
-        nfs_shares               => $_netapp_nfs_shares_sanitized,
-        nfs_shares_config        => $netapp_nfs_shares_config[0],
-        netapp_volume_list       => $netapp_volume_list[0],
-        netapp_vfiler            => $netapp_vfiler[0],
-        netapp_vserver           => $netapp_vserver[0],
-        netapp_controller_ips    => $netapp_controller_ips[0],
-        netapp_sa_password       => $netapp_sa_password[0],
-        netapp_storage_pools     => $netapp_storage_pools[0],
+        netapp_server_hostname  => $netapp_hostname[0],
+        netapp_login            => $netapp_login[0],
+        netapp_password         => $netapp_password[0],
+        netapp_server_port      => $netapp_server_port[0],
+        netapp_storage_family   => $netapp_storage_family[0],
+        netapp_transport_type   => $netapp_transport_type[0],
+        netapp_storage_protocol => $netapp_storage_protocol[0],
+        nfs_shares              => $_netapp_nfs_shares_sanitized,
+        nfs_shares_config       => $netapp_nfs_shares_config[0],
+        netapp_volume_list      => $netapp_volume_list[0],
+        netapp_vfiler           => $netapp_vfiler[0],
+        netapp_vserver          => $netapp_vserver[0],
+        netapp_controller_ips   => $netapp_controller_ips[0],
+        netapp_sa_password      => $netapp_sa_password[0],
+        netapp_storage_pools    => $netapp_storage_pools[0],
       }
-    } elsif str2bool_i("$backend_rbd") {
+    } elsif str2bool_i($backend_rbd) {
       Class['quickstack::ceph::client_packages'] -> Cinder::Backend::Rbd<| |>
       class { '::cinder::volume::rbd':
-        rbd_pool            => $rbd_pool,
-        rbd_ceph_conf       => $rbd_ceph_conf,
+        rbd_pool                                                                  => $rbd_pool,
+        rbd_ceph_conf                                                             => $rbd_ceph_conf,
         rbd_flatten_volume_from_snapshot
-                            => $rbd_flatten_volume_from_snapshot,
-        rbd_max_clone_depth => $rbd_max_clone_depth,
-        rbd_user            => $rbd_user,
-        rbd_secret_uuid     => $rbd_secret_uuid,
+                                                                                           => $rbd_flatten_volume_from_snapshot,
+        rbd_max_clone_depth                                                       => $rbd_max_clone_depth,
+        rbd_user                                                                  => $rbd_user,
+        rbd_secret_uuid                                                           => $rbd_secret_uuid,
```

Comment:
puppet-lint may have suggested the above enormous whitespace, but I would prefer not to see it changed this way.